media-sound/guayadeque/guayadeque-0.4.3.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+WX_GTK_VER=3.0
+
+inherit cmake-utils wxwidgets
+
+DESCRIPTION="Music management program designed for all music enthusiasts"
+HOMEPAGE="http://guayadeque.org/"
+SRC_URI="https://github.com/anonbeat/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ayatana ipod +minimal"
+
+# No test available, Making src_test fail
+RESTRICT="test"
+
+GST_VERSION=1.0
+GST_DEPS="
+	dev-perl/GStreamer
```

Comment:
dev-perl/GStreamer is gstreamer-0.10, in 1.0 world this would be Glib-Object-Introspection + introspection USE deps on what's used. But I don't see any perl anywhere, so just nuke it